config-lite.yml

```diff
@@ -18,6 +18,13 @@ config:
     # If enabled, the proxy will act as a lightweight reverse proxy to support new types of deployment architecture.
     # Default: false
     enabled: true
+    # Maximum number of players to accept, 0 means unlimited.
+    # When this limit is reached, new connections will be rejected.
+    # Default: 0
+    maxPlayers: 0
+    # Custom message to show when the maximum player limit is reached.
+    # Default: §cThis Node is full.Please try another node.
+    maxPlayersMessage: "§cThis Node is full.Please try another node."
```

Comment:
Should use our text config component like other settings do to support the different legacy and modern text types.